WindowsServerDocs/remote/remote-desktop-services/rds_vdi-recommendations-1909.md

```diff
@@ -140,7 +140,7 @@ You might note that this document and the associated scripts on GitHub do not mo
 
 ### Universal Windows Platform (UWP) application cleanup
 
-One of the goals of a VDI image is to be as light as possible. One way to reduce the size of the image is to remove UWP applications that won't be used in the environment. With UWP apps, there are the main application files, also known as the payload. There is a small amount of data stored in each user’s profile for application specific settings. There is also a small amount of data in the ‘All User’s’ profile.
+One of the goals of a VDI image is to be as light as possible. One way to reduce the size of the image is to remove UWP applications that won't be used in the environment. With UWP apps, there are the main application files, also known as the payload. There is a small amount of data stored in each user's profile for application specific settings. There is also a small amount of data in the 'All User's' profile.
```

Comment:
I am fairly sure that the form "All User" (singular) is not a useful grammatical form (maybe even invalid), so we can skip the possessive singular form here:
```suggestion
One of the goals of a VDI image is to be as light as possible. One way to reduce the size of the image is to remove UWP applications that won't be used in the environment. With UWP apps, there are the main application files, also known as the payload. There is a small amount of data stored in each user's profile for application specific settings. There is also a small amount of data in the 'All Users' profile.
```
Even though it is possible to use the possessive form of "All Users" as in "the All Users' profile", it is rarely needed in technical documentation.

Reply:
Indeed, you are correct. I didn't even notice it. The name of the folder is "All Users", so there should never have been an apostrophe there to begin with.